### PR TITLE
Setup Kokoro for Github repo.

### PR DESCRIPTION
**Pull request by wangxf123456**
Test pull request

### CODE REVIEW

File: kokoro/ubuntu/integration_test/presubmit.cfg [hunk 1]
```diff
@@ -0,0 +1,9 @@
+env_vars {
```
Comment (brianquinlan) [inline kokoro/ubuntu/integration_test/presubmit.cfg]: Copyright at top?

File: kokoro/ubuntu/test/continuous.cfg [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Location of the continuous bash script in Git.
```
Comment (brianquinlan) [inline kokoro/ubuntu/test/continuous.cfg]: Copyright at top?

File: kokoro/ubuntu/test/presubmit.cfg [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# Location of the continuous bash script in Git.
```
Comment (brianquinlan) [inline kokoro/ubuntu/test/presubmit.cfg]: Copyright at top?

File: kokoro/ubuntu/integration_test/presubmit.cfg (hunk 1 above)
Comment (wangxf123456) [reply kokoro/ubuntu/integration_test/presubmit.cfg]: Done.

File: kokoro/ubuntu/test/continuous.cfg (hunk 2 above)
Comment (wangxf123456) [reply kokoro/ubuntu/test/continuous.cfg]: Done.

File: kokoro/ubuntu/test/presubmit.cfg (hunk 3 above)
Comment (wangxf123456) [reply kokoro/ubuntu/test/presubmit.cfg]: Done.

**Pull request merged**